src/pages/contracts/[id].vue

```diff
@@ -15,7 +19,7 @@
       </app-link>
     </template>
   </page-header>
-  <template v-if="!isLoading">
+  <template v-if="!isLoading && contractFound">
```

Comment:
```suggestion
  <template v-if="hasError">
```



there should be simple expressions in templates so this should go to computed https://vuejs.org/style-guide/rules-strongly-recommended.html#simple-expressions-in-templates

Reply:
I don't like the result of following this rule in this specific scenario. The following example looks more readable to me:
![image](https://github.com/aeternity/aescan/assets/46789227/2e59162a-a74b-4f9d-bb10-38225ffdcecf)

Rather than scrolling up and down the file to see what logic is hidden behind the proposed computed.

Reply:
It has been simplified in the new version